packages/design-system/postcss.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	plugins: {
+    autoprefixer: {},
```

Comment:
nitpick: Indentation issue here. Probably tabs vs. spaces issue.

Reply:
fixed. can you please approve again :pray: